spec.go

```diff
@@ -177,6 +181,9 @@ func (a *Artifacts) IsEmpty() bool {
 	if len(a.ConfigFiles) > 0 {
 		return false
 	}
+	if len(a.DocFiles) > 0 {
+		return false
+	}
```

Comment:
Does the `LicenseFiles` Field need to be check too if empty?
```suggestion
	}
	if len(a.LicenseFiles) > 0 {
		return false
	}
```

Reply:
oh yes, thanks I forgot! let me update.